modules/drivers/robosense/conf/rs128.pb.txt

```diff
@@ -0,0 +1,15 @@
+model: "RS128"
+frame_id: "rslidar_128"
+ip: "192.168.1.200"
```

Comment:
Why the 128, 32, 16 share the same ip, msop and difop port? If you actually recommend setting them to different ports please differentiate them in default files.

Reply:
We leave the config file as the way it is since all models of RoboSense LiDAR do have the same default settings for ip, msop and difop port?